update-method/src/main/java/com/iluwatar/update/method/App.java

```diff
@@ -0,0 +1,44 @@
+/**
+ * The MIT License
+ * Copyright (c) 2014-2016 Ilkka Seppälä
+ *
+ * Permission is hereby granted, free of charge, to any person obtaining a copy
+ * of this software and associated documentation files (the "Software"), to deal
+ * in the Software without restriction, including without limitation the rights
+ * to use, copy, modify, merge, publish, distribute, sublicense, and/or sell
+ * copies of the Software, and to permit persons to whom the Software is
+ * furnished to do so, subject to the following conditions:
+ *
+ * The above copyright notice and this permission notice shall be included in
+ * all copies or substantial portions of the Software.
+ *
+ * THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS OR
+ * IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY,
+ * FITNESS FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT. IN NO EVENT SHALL THE
+ * AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER
+ * LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM,
+ * OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN
+ * THE SOFTWARE.
+ */
+package com.iluwatar.update.method;
+
+/**
+ * 
+ * In State pattern the container object has an internal state object that defines the current
+ * behavior. The state object can be changed to alter the behavior.
+ * <p>
+ * This can be a cleaner way for an object to change its behavior at runtime without resorting to
+ * large monolithic conditional statements and thus improves maintainability.
+ * <p>
+ * In this example the {@link Mammoth} changes its behavior as time passes by.
+ * 
+ */
+public class App {
+
+  /**
+   * Program entry point
+   */
+  public static void main(String[] args) {
+
+  }
```

Comment:
The main method content is missing. Additionally you should describe the pattern and explain the implementation in the `App` JavaDocs.